src/acl/Acl.cc

```diff
@@ -103,6 +104,22 @@ ACL::FindByName(const char *name)
     return nullptr;
 }
 
+static void
+AddToList(ACL *acl, ACL **head)
```

Comment:
Tests aside, the `head` always points to Config.aclList, right? If you can refactor the unit test a bit, let's not add `head` parameter and just use Config.aclList. In that case, we should also rename this helper function:

```suggestion
RegisterNamed(ACL * const acl)
```